src/lc-braintree/BraintreeForm.jsx

```diff
@@ -76,21 +76,27 @@ class BraintreeForm extends React.Component {
         this.handleChange = this.handleChange.bind(this);
 
         this._notificationSystem = null;
+
+        const emptyBillingAddress = {
```

Comment:
You don't need to create an emptyBillingAddress like this. You can just check if `props.address` is undefined. If so, set it to `{}`.